talent/job-descriptions/applications.md

```diff
@@ -0,0 +1,68 @@
+
+- source{d} raised over $8MM till date and we are currently growing our team. 
+
+- We monetize by deploying our stack and tooling within large tech companies.
+
+## TEAM
+
+- Engineering consists of five different teams, that mimic the architecture of our product:
+
+- Applications (Scala, Go and Python and Frontend tools): Builds CLI/Web applications based on ML research.
+- Machine Learning (C++ and Python): R&D of machine learning on source code.
+- Data Retrieval (Scala and Go): Builds the technology that finds, fetches, stores and analyzes over +60M Git repositories.
+- Language Analysis (Go and another +15): Works on Babelfish, the universal code parsing server.
+- Infrastructure (Go and Python): Manages on-prem bare metal servers with Kubernetes and CoreOS.
+ 
+ 
+We really care about open source, so everything is developed in public, and you can easily check, use and contribute to any libraries and applications built by source{d} engineers on our Github orgs:
+
+- bblfsh/dashboard:  a playground application for Babelfish server, turning code files into Universal Abstract Syntax Trees (UASTs). 
+- src-d/kmcuda:  fast K-means and K-nearest neighbours on NVIDIA GPUs.  
+- src-d/minhashcuda:  fast Weighted MinHash on NVIDIA GPUs. It allows for fast nearly-duplicate repository detection: scan +17 git repositories in 30 minutes to find 1.5M duplicates (the results were published on data.world).  
+- src-d/engine:  a library for running scalable data retrieval pipelines that process any number of Git repositories for source code analysis.  
+- src-d/hercules:  analysis and visualization of a Git repository history. 
+- src-d/gemini: find duplicate code in git repositories.
+
+
+## ROLE
+
+- This position is for a Senior Engineer on the Applications team.
+
+- As an Applications team engineer, you will have a two-fold role:  on one hand it's a Data Engineering, building production-level pipelines to collect and extract data from all git repositories in the world, on another hand it's applied Machine Learning, by productionizing Machine Learning research and wrapping it useful end-user applications.
+
+- You will be expected to have strong backend coding skills in at least one programming language and very good algorithmic skills. Scala coding skills and knowledge about Apache Spark aren't required but will be highly appreciated.
+
+- As you going to be working closely with the Machine Learning team - strong communication skills and interest in Machine Learning, data processing and visualization are more than welcome. 
+
+- You will have to deal with models, built using modern ML stack written in Python and C++ like Tensorflow, Tensorboard, CUDA, so familiarity with any of those tools is a plus.
+
+- Building tools for day to day use by software engineers require good understanding of modern development processes and approaches: Unit/Integration testing, scrum, code reviews, CI/CD, Containers, etc.
```

Comment:
I'd say this is a very important requirement for the offer, and it surprises me that it appears last.